raiden-ts/tests/e2e/raiden.spec.ts

```diff
@@ -837,11 +835,21 @@ describe('Raiden', () => {
         await new Promise((resolve) => setTimeout(resolve, 100));
       });
 
-      afterEach(() => raiden1.stop());
-
       test('fail: direct channel without enough capacity, pfs disabled', async () => {
-        expect.assertions(2);
+        expect.assertions(3);
```

Comment:
What is the background of increasing the node count here and there?

Reply:
If you look closely, the new opened channel has an `expect` on it

Reply:
Ah f*** I can't read and misunderstood that. Super dump. I thought you increased the number of nodes.